doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1595,11 +1599,11 @@ A subscription is represented by a JSON object with the following fields:
 
 A `subject` contains the following subfields:
 
-+ `entities`: A list of objects, each one composed of the following subfields (`id`/`idPattern` or
-  `type` must be present):
-    + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same
-      time, but at least one of them must be present.
-    + `type`: Type of the affected entities (optional).
++ `entities`: A list of objects, each one composed of the following subfields:
+  + `id` or `idPattern`: Id or pattern of the affected entities. Both cannot be used at the same
+     time, but at least one of them must be present.
```

Comment:
Note that the text "Both cannot be used at the same time, but one of them must be present" is already used in the previous version. In the PR, I have just moved it a bit.

However, I understand that your suggestion should be moved to the the fiware-ngsiv2-reference.errata to be included in the "proof read" for the next RC.